### ISSUE TITLE
grid not accessible to screen readers

### ISSUE THREAD

**Issue by mikolysz** [aardappel/treesheets#142]
The grid control is not accessible to screen readers, software used by the blind to navigate the computer. They can read the menus, option windows etc,. but not the main control. I can launch the program and browse throuhg menus, but no other content is accessible. ANy hope of getting this fixed?

**Comment by aardappel** [comment]
I'm afraid the whole grid is custom rendered, meaning that it is not visible as text to the operating system (same as other customly rendered apps like most games). It will be a ton of work to make that properly accessible, I can't promise that will happen any time soon.
